### PR TITLE
feat: Added guide to query alerts for quickstarts

### PR DESCRIPTION
**Pull request by mehreentahir16**
<sup> Created from VS Code using [CodeStream](https://codestream.com/?utm_source=cs&utm_medium=pr&utm_campaign=github*com)</sup>

### CODE REVIEW

**Comment by alexronquillo** [comment]
test

```
nr-book
```
								
[src/data/nav.yml](https://github.com/newrelic/developer-website/blob/322e5923eb22f187bdbac9ee5f78779c61a19c68/src/data/nav.yml#L194) (Line 194)

File: src/markdown-pages/contribute-to-quickstarts/query-alerts-for-quickstart.mdx [hunk 1]
```diff
@@ -0,0 +1,308 @@
...
+# Time in seconds; 300 - 2592000 (Default: 86400 [1 day])
+violationTimeLimitSeconds: 2592000
+
+```
+
+Here, you added an outlier alert to your quickstart. 
+
+## Summary
+
+This guide teaches you to query your existing alert conditions using Nerdgraph and use the json response to populate corresponding alert files in quickstart.
```
Comment (alexronquillo) [inline src/markdown-pages/contribute-to-quickstarts/query-alerts-for-quickstart.mdx:306]: You're at the end now, so you can switch to past tense:


> In this guide, you learned...

File: src/markdown-pages/contribute-to-quickstarts/query-alerts-for-quickstart.mdx [hunk 2]
```diff
@@ -0,0 +1,308 @@
...
+
+```
+
+Here, you added an outlier alert to your quickstart. 
+
+## Summary
+
+This guide teaches you to query your existing alert conditions using Nerdgraph and use the json response to populate corresponding alert files in quickstart. 
+If you're not familiar with alerts, then check our [docs for alerts](https://docs.newrelic.com/docs/alerts-applied-intelligence/).
+To learn more about the quickstarts and how to create one, please refer to [Build a quickstart](/contribute-to-quickstart/build-a-quickstart).
```
Comment (alexronquillo) [inline src/markdown-pages/contribute-to-quickstarts/query-alerts-for-quickstart.mdx:308]: Could you make a list here?


> In this guide, you learned how to query your existing alert conditions using NerdGraph and how to use the query's JSON response to populate corresponding alert files in quickstart. Read our documentation to learn more about:
>
> - [Alerts](https://docs.newrelic.com/docs/alerts-applied-intelligence/)
> - [Building quickstarts](/contribute-to-quickstart/build-a-quickstart)

**Review by alexronquillo — CHANGES_REQUESTED** [review]
Thanks @mehreentahir16 I think this is a great start! A lot of this feedback is about whether we have the right information in the right places. The rest is about language-level details, such as preferring present tense, active voice, etc. Looking forward to seeing your next pass!

File: src/markdown-pages/contribute-to-quickstarts/query-alerts-for-quickstart.mdx [hunk 3]
```diff
@@ -0,0 +1,308 @@
...
+  }
+}
+
+```
+
+The above query will return you the json representation of the specified static condition that you can use to update your quickstart. 
+
+![static alert query](../../images/query-alerts/static-alert-query.png)
+
+Use the _static-alert.yml_ file from quickstart template and start populating it using the information returned from above query.
```
[the remark this answers is not in the document]
Comment (mehreentahir16) [reply src/markdown-pages/contribute-to-quickstarts/query-alerts-for-quickstart.mdx:100]: We're assuming that user already has quickstart. If they do, then they would have covered this part already...cloning the GitHub repo, copy the template for dashboard and alerts. 

> What if the user is working with a quickstart that doesn't even have an alerts directory? etc.

The provided template for quickstart contains the directory for alerts. This assumption doesn't seem valid to me unless the narrative is to extend an already existing quickstart.

**Pull request merged**